### PR TITLE
bioperl testing

### PR DESCRIPTION
**Pull request by bgruening**

### CODE REVIEW

**Comment by erasche** [comment]
> Can't locate LWP/UserAgent.pm in @INC (you may need to install the LWP::UserAgent module) (

???? 

That should be part of the bioperl dep tree?

**Comment by bgruening** [comment]
It's not at least not in bioconda. It tried to work on this but it is a huge pain as it touches millions of perl deps: https://github.com/bioconda/bioconda-recipes/compare/perl_html?expand=1

**Comment by erasche** [comment]
woah. Yuck. :( what to do about this?

**Comment by bgruening** [comment]
I guess I need to fix the conda package. But this will take some time.

**Comment by bgruening** [comment]
Btw. @erasche I would be fine to put all tools automatically to all TS that are under conda testing. What do you think?

**Comment by erasche** [comment]
@bgruening absolutely, tests pass = should be in TS, in my opinion. If it's on master, we've all agreed we're happy with the software.

**Comment by bgruening** [comment]
This is for Eric! He probably knows what this means for JBrowse :)

@erasche I needed t change the test data.
I will leave this to you to merge as the conda version is different from the TS version.

File: packages/package_boost_1_55/tool_dependencies.xml [hunk 1]
```diff
@@ -23,6 +23,7 @@
                     <environment_variable name="BOOST_ROOT_DIR" action="set_to">$INSTALL_DIR/boost</environment_variable>
                     <environment_variable action="prepend_to" name="LD_LIBRARY_PATH">$INSTALL_DIR/boost/lib/</environment_variable>
                     <environment_variable action="prepend_to" name="LIBRARY_PATH">$INSTALL_DIR/boost/lib/</environment_variable>
+                    <environment_variable action="prepend_to" name="LIBRARY_PATH">$INSTALL_DIR/boost/lib/</environment_variable>
```
Comment (erasche) [inline packages/package_boost_1_55/tool_dependencies.xml]: is this a duplicate of the above line?

**Comment by erasche** [comment]
@bgruening <3 <3 <3

That changed test data looks fine to me.

File: packages/package_boost_1_55/tool_dependencies.xml (hunk 1 above)
Comment (bgruening) [inline packages/package_boost_1_55/tool_dependencies.xml]: Not sure why this is here in first place? Bad rebase? Will fix it.

**Comment by bgruening** [comment]
Your welcome @erasche! :)
I fixed that one liner.

**Pull request merged**